sentences/fr/media_player_HassMediaPause.yaml

```diff
@@ -0,0 +1,9 @@
+language: fr
+intents:
+  HassMediaPause:
+    data:
+      - sentences:
+          - "mets [<le>]{name} sur pause"
+          - "Pause sur [<le>]{name}"
```

Comment:
Same comment as for `HassSetPosition` : Verbs should be expansion rules and support infinitive forms